<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssistantGUI</class>
 <widget class="QWidget" name="AssistantGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AssistantGUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Current:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentMaterial_label">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="save_pushButton">
         <property name="text">
          <string>Save current material</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="next_pushButton">
         <property name="text">
          <string>Next material</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterMaterials_pushButton">
         <property name="text">
          <string>Filter by size/microfragments</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="undo_pushButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redo_pushButton">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="displayAssistantList_pushButton">
         <property name="text">
          <string>Show assistant List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openList_pushButton">
         <property name="text">
          <string>Open assistant list in external app</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
